src/main.rs

```diff
@@ -46,6 +46,14 @@ async fn main() -> Result<(), Box<dyn Error>> {
         std::env::var("SLEEP_BETWEEN_REQUESTS").expect("SLEEP_BETWEEN_REQUESTS must be set.");
     let sleep_between_requests = sleep_between_requests.as_str().parse().unwrap();
 
+    let check_mentions = std::env::var("IGNORE_MENTIONS")
```

Comment:
```suggestion
    let check_mentions = std::env::var("CHECK_MENTIONS")
```

Rename the env var to `CHECK_MENTIONS` please